src/analysis/gMCS/GPR2models.m

```diff
@@ -71,8 +66,8 @@
 
 % Step 2: Create models
 disp('Calculating Networks for GPR rules...');
-% parfor (i=1:length(selected_rxns),poolsize)
-for i=1:length(selected_rxns)
+parfor (i=1:length(selected_rxns),poolsize)
```

Comment:
There is a check some lines adove, from line 59 to 65

Reply:
Sorry, didn't see that. But this line `% for i=1:length(selected_rxns)` can then be removed? 

Could you implement a `numWorkers` argument instead of using `p.NumWorkers`?

Reply:
I have removed `% for i=1:length(selected_rxns)`

 `numWorkers ` argument will be implemented for Cplex maximum number of workers